save_features.py

```diff
@@ -18,7 +19,31 @@
 
 device = torch.device('cuda:0' if torch.cuda.is_available() else 'cpu')
 
-def save_features(model, data_loader, outfile ):
+def save_features(model, data_loader, outfile, finetune_iterations):
+    # TODO: number_classes
+    number_classes = -99
```

Comment:
Do you have a good idea how we can determine the number of classes?

Reply:
In relationnet.py>>RelationModule the last linear layer always has one output node. So I guess they evaluate one vs. all others for a new class.

Reply:
True. Should we also add another fc layer? https://github.com/Baschdl/CloserLookFewShot/blob/26c2e25a8dc2a7b3fea26eb7bd2814c3df6208dd/methods/relationnet.py#L143

Reply:
Did you also find the conversion to the binary problem? I guess that should happen in the datamgr and not `save_features.py`